### DISCUSSION TITLE
关于useMemoizedFn的this的一些疑惑

### DISCUSSION THREAD

**Discussion by hanzc0106** [alibaba/hooks#1560] [Q&A] [answered]
https://github.com/alibaba/hooks/blob/2d1cc85e6ef0ae4a32f2115035a87a1bb643a17f/packages/hooks/src/useMemoizedFn/index.ts#L25

这里memoizedFn.current 返回的函数，参数列表是(this, ...args), 这种情况下包装函数如fn(a, b, c), 在调用时实际效果是否会变成fn(b, c)?

**Comment by brickspert** [comment, marked as answer]
参考：https://github.com/alibaba/hooks/pull/1464
